src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,9 @@
-export const UserInfo = () => <>Put the user here</>;
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <>
```

Comment:
you can remove teg <> </>, but I`m not sure

Reply:
Yes, let's remove it